packages/react-lowcode/src/codegen/generation/generators/list/mui-datatable-generator.ts

```diff
@@ -254,6 +255,63 @@ export default class MuiDataTableGenerator implements TableGenerator
       }else return undefined
     }
 
+    generateTablePage(template: string): PageComponent | undefined {
```

Comment:
This does not seems to be Material UI specific code. What about moving this logic to different file?

Reply:
In meantime whole this was moved to its own file - template-resolver.ts